control_plane/src/endpoint.rs

```diff
@@ -488,6 +493,13 @@ impl Endpoint {
             pageserver_connstring: Some(pageserver_connstring),
             safekeeper_connstrings,
             storage_auth_token: auth_token.clone(),
+            // TODO FIXME: This is a hack to test custom extensions locally.
```

Comment:
@lubennikovaav should we get rid of this?
but then our regression tests will fail

Reply:
yep, we should remove this.
and delete the python test of custom ext as well

we'll replace it with e2e tets

Reply:
ok!

Reply:
done